lib/backends/webgl/webgl-context.ts

```diff
@@ -464,4 +468,64 @@ export class WebGLContext {
       }
     }
   }
+
+  beginTimer(): WebGLQuery {
+    if (this.version === 2) {
+      const gl2 = this.gl as WebGL2RenderingContext;
+      const ext = this.disjointTimerQueryWebgl2Extension;
+
+      const query = gl2.createQuery() as WebGLQuery;
+      gl2.beginQuery(ext.TIME_ELAPSED_EXT, query);
+      return query;
+    } else {
+      // TODO: add webgl 1 handling.
+      throw new Error(`Not support WebGL 2`);
```

Comment:
>`Not support WebGL 2` [](start = 22, length = 21)

Maybe we can rephrase this error message to be more accurate, as this is not about WebGL2. How about "WebGL1 profiling currently not supported"?

Same for the error messages below.